src/pages/events/dialogs/new.tsx

```diff
@@ -188,6 +194,13 @@ export const EventNewIncidentDialog: React.FC<EventNewIncidentDialogProps> = ({
       const packed = packIncident(incident);
       mutate(packed, {
         onSuccess: () => {
+          // Do not reset match, for ease of use.
+          setTeam(null);
+
+          setIncidentField("team", null);
+          setIncidentField("notes", "");
+          setIncidentField("rules", []);
+          setIncidentField("outcome", IncidentOutcome.Minor);
```

Comment:
After submitting a violation, make sure to not automatically set the same values again for the next violation.